mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -35,82 +35,87 @@ class _WalletScreenState extends State<WalletScreen> {
     );
 
     return Scaffold(
-      body: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 20),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: [
-            ExpansionPanelList(
-              children: [
-                ExpansionPanel(
-                  headerBuilder: (BuildContext context, bool isExpanded) {
-                    return Row(
-                      children: [
-                        // https://stackoverflow.com/a/70192038 - do not know if this is principled
-                        const SizedBox(
-                            width:
-                                64), // ExpansionPanelList IconContainer size: end margin 8 + padding 16*2 + size 24),
-                        Expanded(
-                          child: Center(
-                              child: AmountText(
-                                  amount: walletChangeNotifier.total(),
-                                  textStyle: const TextStyle(
-                                      fontSize: 20.0, fontWeight: FontWeight.bold))),
-                        )
-                      ],
-                    );
-                  },
-                  body: Padding(
-                    padding: const EdgeInsets.only(left: 8.0, right: 8.0, bottom: 16.0),
-                    child: Column(
-                      children: const [
-                        Padding(
-                          padding: EdgeInsets.symmetric(horizontal: 8.0),
-                          child: BalanceRow(walletType: WalletHistoryItemDataType.lightning),
-                        ),
-                        Padding(
-                          padding: EdgeInsets.symmetric(horizontal: 8.0),
-                          child: BalanceRow(walletType: WalletHistoryItemDataType.onChain),
-                        )
-                      ],
+      body: RefreshIndicator(
+        onRefresh: () async {
+          await walletChangeNotifier.refreshWalletInfo();
```

Comment:
🔧 `refresh_wallet_info` is not doing much! It is only reconstructing the wallet info from the existing information. You will unlikely get new results by doing that, unless we also run a wallet sync on the refresh wallet info. 

Note, after a wallet sync we automatically push a refreshed wallet info to the client. so refreshing it manually will most likely never yield any new results until the sync happened.

Reply:
Oh, I missed that this was a PR to the transaction history which added the sync to the wallet refresh. I only looked at your changes. 😊